### ISSUE TITLE
high light as error for memory export

### ISSUE THREAD

**Issue by linux-china** [Pluralia/web-assembly-plugin#9]
The following code  is correct for wat, and now it's marked as error for '2'. 

```
(module
  (memory $memory (export "memory") 2)
)
```

**Comment by anstarovoyt** [comment]
@linux-china Hi! Thank you for the report. Unfortunately, the specification isn't clear enough in [this place](https://webassembly.github.io/spec/core/text/modules.html#memories).
It states that ``memory``  can be defined with exports
```
‘(’ ‘𝚖𝚎𝚖𝚘𝚛𝚢’  𝚒𝚍?  ‘(’ ‘𝚎𝚡𝚙𝚘𝚛𝚝’  𝚗𝚊𝚖𝚎 ‘)’  … ‘)’
```
and 
>The latter abbreviation can be applied repeatedly, with “…” containing another import or export or an inline data segment.

And it doesn't mention the ``𝚖𝚎𝚖𝚝𝚢𝚙𝚎`` that is used as a limit for ``memory`` instructions. 

I will try to fix the issue next week.

**Comment by anstarovoyt** [comment]
@linux-china fixed in 1.2
